#include <iostream>
#include "myexception.h"

using namespace std;

int main()
{
    throw
    return 0;
}
